src/main/java/com/openlattice/chronicle/configuration/TwilioConfiguration.kt

```diff
@@ -0,0 +1,32 @@
+package com.openlattice.chronicle.configuration
+
+import com.fasterxml.jackson.annotation.JsonCreator
+import com.fasterxml.jackson.annotation.JsonIgnore
+import com.fasterxml.jackson.annotation.JsonProperty
+import com.kryptnostic.rhizome.configuration.Configuration
+import com.kryptnostic.rhizome.configuration.ConfigurationKey
+import com.kryptnostic.rhizome.configuration.SimpleConfigurationKey
+import com.kryptnostic.rhizome.configuration.annotation.ReloadableConfiguration
+
+private const val CONFIG_FILE_NAME = "twilio.yaml"
+@ReloadableConfiguration(uri = CONFIG_FILE_NAME)
+
+class TwilioConfiguration @JsonCreator constructor(
```

Comment:
Our conventions are to not have an empty line between annotation and the thing being annotated.

```suggestion
private const val CONFIG_FILE_NAME = "twilio.yaml"

@ReloadableConfiguration(uri = CONFIG_FILE_NAME)
class TwilioConfiguration @JsonCreator constructor(
```

Reply:
Separately, I realize I didn't comment about converting this to a data class in my previous review. This should also be a data class like with `MessageOutcome` and `MessageDetails`. You also don't need the `@JsonCreator constructor` bit.

```
data class TwilioConfiguration(
```